### ISSUE TITLE
Reconcile a11y considerations in Alert and Alert - Expandable

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#1700]
### Duplicate check

- [X] I've searched for any related issues and avoided creating a duplicate issue.

### This update is for:

Component

### What is the name?

Alert

### What is the nature of this update?

- [ ] How to build this component/pattern
- [ ] When to use this component/pattern
- [ ] When to use something else
- [ ] Usability guidance
- [X] Accessibility
- [ ] Implementation
- [ ] Research insights
- [ ] Package information
- [ ] Addition to Forms Library documentation
- [ ] Update to existing Forms Library documentation

### What problem does this solve?

The Alert - Expandable a11y guidance on alt text is more specific than it is for Alert. This should be reconciled.

### Additional Context

This came from a review of our Alert a11y considerations by Julie Stothman, who is helping us out.
